.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,17 @@
+Link to the milestone on Github e.g. https://github.com/nimblehq/git-templates/milestone/41?closed=1
```

Comment:
Should we modify this section as well?

Reply:
Same as https://github.com/nimblehq/infrastructure-templates/pull/21#discussion_r870904993